erpnext/accounts/report/trial_balance/trial_balance.js

```diff
@@ -72,6 +72,13 @@ frappe.require("assets/erpnext/js/financial_statements.js", function() {
 				"fieldtype": "Link",
 				"options": "Finance Book",
 			},
+			{
+				"fieldname": "presentation_currency",
+				"label": __("Currency"),
+				"fieldtype": "Select",
+				"options": erpnext.get_presentation_currency_list(),
+				"default": frappe.defaults.get_user_default("currency")
```

Comment:
By default it should show in Company currency and not in user currency. This can be removed

Reply:
Ok removing